.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Example issue: https://github.com/sbueringer/cluster-api/issues/80

Reply:
Note to myself: update example issue with links to the documentation on this PR

Reply:
Added the following to PR description:

Issue creation: https://github.com/sbueringer/cluster-api/issues/new/choose
Example release tracking issue: https://github.com/sbueringer/cluster-api/issues/80